k8s/superset/chart/superset/templates/postgresql-service.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name }}-postgresql-service
+  labels:
+    app.kubernetes.io/name: "{{ .Release.Name}}"
```

Comment:
Please watch the standards. In line 4 you use `{{ .Release.Name }}`, in line 6: `{{ .Release.Name}` . Keep it the more readable as possible.

Reply:
Check here and everywhere